pkg/tests/ossm/smoke_test.go

```diff
@@ -61,33 +61,21 @@ func TestSmoke(t *testing.T) {
 		t.LogStep("Install bookinfo pods and sleep pod")
 		app.InstallAndWaitReady(t, app.Bookinfo(ns), app.SleepNoSidecar(ns))
 
-		t.NewSubTest(fmt.Sprintf("install bookinfo with smcp %s", fromVersion)).Run(func(t TestHelper) {
+		t.LogStepf("Install SMCP %s and verify it becomes ready", fromVersion)
+		assertSMCPDeploysAndIsReady(t, fromVersion)
```

Comment:
I'd move this into the upgrade subtest, because that's the only place where the old control plane is needed.